docs/media-seek-backward-button.md

```diff
@@ -7,7 +7,11 @@ Button to jump back 30 seconds in the media.
 
 # Attributes
 
-_None_
+# Attributes
+
+| Name          | Type     | Default Value | Description                                                         |
+| ------------- | -------- | ------------- | ------------------------------------------------------------------- |
+| `seek-offset` | `number` | `30`          | Adjusts how much time (in seconds) the playhead should seek forward |
```

Comment:
If this is always seconds, should we just call it `seek-seconds`?

I started looking into a frame advancing button and considered building it on seek forward/backward, which would affect this, but I ultimately felt like that deserved its own button, at least because the default icon would be different.

Reply:
I'm gonna keep it as is. Most of the properties/attrs don't bake in units in their names in DOM-land, so this feels more consistent with those norms/conventions.